lib/commands/test/karma.js

```diff
@@ -64,6 +64,8 @@ module.exports = {
       })
       .option('karma.browsers', { type: 'array', hidden: true }) // defined but hidden so yargs will parse as an array
       .option('karma.reporters', { type: 'array', hidden: true })
+      .option('watch', { type: 'boolean', describe: 'Watch test files for changes and run tests automatically when changes are saved.', alias: 'karma.watch' })
```

Comment:
"Watch test files for changes..." could be ambiguous. At first, I read it to mean "watch only files containing tests, e.g. stuff in ./tests/ or ./src/**/*.test.js" but on review I think it means "watch the files under test for changes, which probably means everything in ./src/ and ./tests/ so that changes to the source and changes to the tests both result in tests re-running." What do you think, could this be clearer?